src/esp/metadata/attributes/ObjectAttributes.cpp

```diff
@@ -54,19 +54,19 @@ std::string AbstractObjectAttributes::getObjectInfoInternal() const {
       .append(1, ',')
       .append(getCollisionAssetHandle())
       .append(1, ',')
-      .append(cfg.value("scale"))
+      .append(getAsString("scale"))
       .append(1, ',')
-      .append(cfg.value("margin"))
+      .append(std::to_string(getMargin()))
       .append(1, ',')
-      .append(cfg.value("orient_up"))
+      .append(getAsString("orient_up"))
       .append(1, ',')
-      .append(cfg.value("orient_front"))
+      .append(getAsString("orient_front"))
       .append(1, ',')
-      .append(cfg.value("units_to_meters"))
+      .append(std::to_string(getUnitsToMeters()))
       .append(1, ',')
-      .append(cfg.value("friction_coefficient"))
+      .append(std::to_string(getFrictionCoefficient()))
       .append(1, ',')
-      .append(cfg.value("restitution_coefficient"))
+      .append(std::to_string(getRestitutionCoefficient()))
       .append(1, ',')
       .append(getCurrShaderTypeName())
       .append(1, ',')
```

Comment:
Same here, `formatString()` or [`Utility::String::join()`](https://doc.magnum.graphics/corrade/namespaceCorrade_1_1Utility_1_1String.html#a9af3018407a9cac247c09da9ac8e577d) at least :P

Reply:
I'm going to strip out all of the offending functionality from this PR and refactor all of these to use the formatString process in another PR.